MapboxNavigation/RouteVoiceController.swift

```diff
@@ -207,3 +212,7 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate, AVAudioP
         speechSynth.speak(utterance)
     }
 }
+
+@objc public protocol VoiceControllerDelegate {
+    @objc optional func spokenInstructionsDidFail(_ voiceController: RouteVoiceController, error: Error)
```

Comment:
Delegate methods should always accept the delegator object as the first parameter and be named based on the delegator object. So this method should be `voiceController(_:spokenInstructionsDidFailWith:)`. Also, the Objective-C signature needs to include the nouns: `-voiceController:spokenInstructionsDidFailWithError:`.